db/migrate/20230120124639_create_return_details.rb

```diff
@@ -0,0 +1,12 @@
+class CreateReturnDetails < ActiveRecord::Migration[7.0]
+  def change
+    create_table :return_details, id: :uuid do |t|
+      t.string :reason_type, null: false
+      t.string :reason_text, null: false
```

Comment:
Wondering if reason text is needed? Is it the "human text" of the reason type identifier?
If so, perhaps is best to let each consumer of the API (apply/review) use the identifier and locales to translate to human text the reason?

This may be useful if we implement welsh, where a provider might want to read the reason in welsh.

Reply:
I thought it best to capture the text at the time of submission in case it changed in the locale and we needed to know the exact working as selected by the caseworker. (The return_reason key could still be used on apply to load from I18n for welsh etc.)
Happy to remove if I'm being too cautious!